packages/web-react/src/components/FileUploader/README.md

```diff
@@ -0,0 +1,383 @@
+# FileUploader
+
+The FileUploader implementation is used to select files either by selecting files from the device itself,
+or by drag and drop if the device supports it. React package extends [web package][FileUploader].
```

Comment:
`React package extends [web package][FileUploader].` Please check out for example Alert Readme. This link is added to the end of the file like this `For detailed information see [Alert](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/Alert/README.md) component`.

Reply:
?

Reply:
I mean the copy. I think we do this the same way in all readmes.

Reply:
Done